docs/next/guides/sidebar.js

```diff
@@ -1,189 +1,189 @@
 const gettingStartedItems = [
-  'guides/getting-started/introduction',
-  'guides/getting-started/demo',
-  'guides/getting-started/installation',
-  'guides/getting-started/binding-to-data',
-  'guides/getting-started/saving-data',
-  'guides/getting-started/setting-options',
-  'guides/getting-started/grid-size',
-  'guides/getting-started/events-and-hooks',
-  'guides/getting-started/license-key',
+  { link: 'guides/getting-started/introduction' },
+  { link: 'guides/getting-started/demo' },
+  { link: 'guides/getting-started/installation' },
+  { link: 'guides/getting-started/binding-to-data' },
+  { link: 'guides/getting-started/saving-data' },
+  { link: 'guides/getting-started/setting-options' },
+  { link: 'guides/getting-started/grid-size' },
+  { link: 'guides/getting-started/events-and-hooks' },
+  { link: 'guides/getting-started/license-key' },
 ];
 
 const integrateWithReactItems = [
-  'guides/integrate-with-react/react-installation',
-  'guides/integrate-with-react/react-simple-example',
-  'guides/integrate-with-react/react-modules',
-  'guides/integrate-with-react/react-hot-column',
-  'guides/integrate-with-react/react-setting-up-a-language',
-  'guides/integrate-with-react/react-custom-context-menu-example',
-  'guides/integrate-with-react/react-custom-editor-example',
-  'guides/integrate-with-react/react-custom-renderer-example',
-  'guides/integrate-with-react/react-language-change-example',
-  'guides/integrate-with-react/react-redux-example',
-  'guides/integrate-with-react/react-hot-reference',
+  { link: 'guides/integrate-with-react/react-installation' },
```

Comment:
As an example, can you skip the React section in the JavaScript build of the docs?

Reply:
Do you mean `{ link: 'guides/integrate-with-react/react-installation' },` page? GitHub shows the suggestion for 14th line.

Reply:
I just added the comment on this line to create a thread. Let's continue the discussion in the next thread (below).